day_two_guide/topics/api_service_status.adoc

```diff
@@ -6,28 +6,22 @@ Module included in the following assemblies:
 * day_two_guide/environment_health_checks.adoc
 ////
 
-The OpenShift API service, `atomic-openshift-master-api.service`, runs on all
-master instances. To see the status of the service:
+The OpenShift API service runs on all
+master instances. To see the status of the service,
+view the pods in the *kube-system* project:
 
 ----
-$ systemctl status atomic-openshift-master-api.service
-● atomic-openshift-master-api.service - Atomic OpenShift Master API
-   Loaded: loaded (/usr/lib/systemd/system/atomic-openshift-master-api.service; enabled; vendor preset: disabled)
-   Active: active (running) since Thu 2017-11-30 11:40:19 EST; 5 days ago
-     Docs: https://github.com/openshift/origin
- Main PID: 30047 (openshift)
-   Memory: 65.0M
-   CGroup: /system.slice/atomic-openshift-master-api.service
-           └─30047 /usr/bin/openshift start master api --config=/etc/origin/master/ma...
+$ oc get pod -n kube-system
 
-Dec 06 09:15:49 ocp-master-94zd atomic-openshift-master-api[30047]: I1206 09:15:49.85...
-Dec 06 09:15:50 ocp-master-94zd atomic-openshift-master-api[30047]: I1206 09:15:50.96...
-Dec 06 09:15:52 ocp-master-94zd atomic-openshift-master-api[30047]: I1206 09:15:52.34...
+NAME                             READY     STATUS    RESTARTS   AGE
+master-api.example.com           1/1       Running   0          56d
+master-controllers.example.com   1/1       Running   0          56d
+master-etcd.example.com          1/1       Running   0          56d
 ----
 
 The API service exposes a health check, which can be queried externally with:
 
 ----
-$ curl -k https://master.example.com/healthz
+$ curl -k https://master-api.example.com/healthz
 ok
```

Comment:
It might not be true. The `master-api.example.com` looks like the pod name and you cannot curl it. You should get the hostname or IP of the node which the pod landed then curl the host. for example:
```
$ oc get pod -n kube-system -o wide
NAME                                               READY     STATUS    RESTARTS   AGE       IP            NODE
master-api-qe-anli310master-etcd-zone1-1           1/1       Running   0          7h        10.240.0.16   qe-anli310master-etcd-zone1-1
$ curl -k https://qe-anli310master-etcd-zone1-1/healthz
ok
```